test/integration/full/configuration/tag-exclude.js

```diff
@@ -0,0 +1,102 @@
+describe('all rules test', () => {
+  const experimentalRuleId = 'label-content-name-mismatch';
```

Comment:
This relies on the current state of axe and rules that could be made non-experimental or removed eventually. It would be better to configure axe with custom rules that are experimental and deprecated and test those ones instead.